charts/fluentd-elasticsearch/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: fluentd-elasticsearch
-version: 6.2.4
+version: 6.2.5
```

Comment:
new feature. therefore version should be 6.3.0

Reply:
Cool, fixed.